src/request/derive-polygon-address/index.html

```diff
@@ -0,0 +1,102 @@
+    <script defer bundle-polygon src="../../lib/polygon/PolygonKey.js"></script>
+
+    <script defer src="DerivePolygonAddress.js"></script>
+    <script defer src="DerivePolygonAddressApi.js"></script>
+    <script defer src="index.js"></script>
+
+    <link rel="stylesheet" bundle-toplevel href="../../../node_modules/@nimiq/style/nimiq-style.min.css">
+    <link rel="stylesheet" bundle-toplevel href="../../nimiq-style.css">
+    <link rel="stylesheet" bundle-toplevel href="../../common.css">
+    <link rel="stylesheet" bundle-toplevel href="../../components/PasswordInput.css">
+    <link rel="stylesheet" bundle-toplevel href="../../components/PasswordBox.css">
+
+    <link rel="stylesheet" href="./DerivePolygonAddress.css">
+</head>
+<body class="loading">
+    <div class="header-top">
+        <div class="nimiq-app-name">
+            <span class="nimiq-logo"></span>
+        </div>
+    </div>
+
+    <div id="app">
+        <div id="unlock" class="page nq-card">
+            <div class="page-body nq-card-body">
+                <div class="img">
+                    <img src="../../assets/icons/usdc-activation.svg" class="usdc-activation-logo">
+                </div>
+
+                <div class="flex-grow"></div>
+
+                <h1 class="nq-h1" data-i18n="derive-polygon-address-heading">Activate USDC</h1>
+                <p class="nq-text" data-i18n="derive-polygon-address-text">
+                    USDC is a stablecoin pegged to USD and now available in the wallet.
+                </p>
+
+                <div class="flex-grow"></div>
+            </div>
+
+            <div class="page-footer nq-card-footer">
+                <form class="password-box"></form>
+            </div>
+        </div>
+
+        <button class="global-close nq-button-s display-none">
+            <svg class="nq-icon"><use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-arrow-left-small"/></svg>
+            <span id="global-close-text"></span>
+        </button>
+
+        <div id="loading" class="page">
+            <!-- Loading spinner SVG -->
+            <svg height="48" width="54" color="inherit" class="loading-spinner"><g>
+                <path class="big-hex" d="M51.9,21.9L41.3,3.6c-0.8-1.3-2.2-2.1-3.7-2.1H16.4c-1.5,0-2.9,0.8-3.7,2.1L2.1,21.9c-0.8,1.3-0.8,2.9,0,4.2 l10.6,18.3c0.8,1.3,2.2,2.1,3.7,2.1h21.3c1.5,0,2.9-0.8,3.7-2.1l10.6-18.3C52.7,24.8,52.7,23.2,51.9,21.9z" stroke="currentColor" stroke-width="3" fill="none" stroke-linecap="round" opacity="0.4" stroke-dasharray="92.5 60"/>
+                <path class="small-hex" d="M51.9,21.9L41.3,3.6c-0.8-1.3-2.2-2.1-3.7-2.1H16.4c-1.5,0-2.9,0.8-3.7,2.1L2.1,21.9c-0.8,1.3-0.8,2.9,0,4.2 l10.6,18.3c0.8,1.3,2.2,2.1,3.7,2.1h21.3c1.5,0,2.9-0.8,3.7-2.1l10.6-18.3C52.7,24.8,52.7,23.2,51.9,21.9z" stroke="currentColor" stroke-width="3" fill="none" stroke-linecap="round" stroke-dasharray="47.5 105"/>
+            </g></svg>
```

Comment:
The same attributes on a `<path/>` can be applied directly on the parent group `<g></g>` to prevent repetition
```suggestion
            <svg height="48" width="54" color="inherit" class="loading-spinner"><g stroke-width="3" fill="none" stroke-linecap="round" stroke="currentColor">
                <path class="big-hex" d="M51.9,21.9L41.3,3.6c-0.8-1.3-2.2-2.1-3.7-2.1H16.4c-1.5,0-2.9,0.8-3.7,2.1L2.1,21.9c-0.8,1.3-0.8,2.9,0,4.2 l10.6,18.3c0.8,1.3,2.2,2.1,3.7,2.1h21.3c1.5,0,2.9-0.8,3.7-2.1l10.6-18.3C52.7,24.8,52.7,23.2,51.9,21.9z" opacity="0.4" stroke-dasharray="92.5 60"/>
                <path class="small-hex" d="M51.9,21.9L41.3,3.6c-0.8-1.3-2.2-2.1-3.7-2.1H16.4c-1.5,0-2.9,0.8-3.7,2.1L2.1,21.9c-0.8,1.3-0.8,2.9,0,4.2 l10.6,18.3c0.8,1.3,2.2,2.1,3.7,2.1h21.3c1.5,0,2.9-0.8,3.7-2.1l10.6-18.3C52.7,24.8,52.7,23.2,51.9,21.9z" stroke-dasharray="47.5 105"/>
            </g></svg>
```